### PR TITLE
support shorthand syntax in example generation

### PR DESCRIPTION
**Pull request by necusjz**

### CODE REVIEW

File: src/aaz_dev/swagger/controller/_shorthand.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+def serialize(obj):
```
Comment (kairu-ms) [inline src/aaz_dev/swagger/controller/_shorthand.py]: The shorthand syntax should be belong to Command module not Swagger module.

File: src/aaz_dev/swagger/controller/_example_builder.py [hunk 2]
```diff
@@ -106,7 +106,7 @@ def mapping(self, example_dict):
                 val=value
             )
             if item and item.is_top_level:
-                self.example_items.append((item.arg_option, json.dumps(value)))
+                self.example_items.append((item.arg_option, serialize(value)))
```
Comment (kairu-ms) [inline src/aaz_dev/swagger/controller/_example_builder.py]: The final output serialization no matter to json string or shorthand syntax string, should be implemented in Command module.

File: src/aaz_dev/swagger/controller/_shorthand.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+        else:
+            escaped = str(obj)
+            escaped = escaped.replace("'", "'/")  # use '/ to input '
+
+            # with space, null/help expressions or other special characters
+            if any(char in escaped for char in (" ", "null", "??", ":", ",", "{", "}", "[", "]")):
+                escaped = "'" + escaped + "'"
+
+            return repr(escaped)[1:-1]  # ignore quotes
```
Comment (kairu-ms) [inline src/aaz_dev/swagger/controller/_shorthand.py]: Should replace `None` by `null` in shorthand syntax expression

File: src/aaz_dev/swagger/controller/_shorthand.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        elif isinstance(obj, list):
+            return "[" + ",".join(dfs(i) for i in obj) + "]"
+
+        else:
+            escaped = str(obj)
+            escaped = escaped.replace("'", "'/")  # use '/ to input '
+
+            # with space, null/help expressions or other special characters
+            if any(char in escaped for char in (" ", "null", "??", ":", ",", "{", "}", "[", "]")):
+                escaped = "'" + escaped + "'"
```
Comment (kairu-ms) [inline src/aaz_dev/swagger/controller/_shorthand.py:15]: actually the escape should be far more complicated than that. You can reference this [doc](https://github.com/Azure/azure-cli/blob/dev/doc/shorthand_syntax.md#use--to-pass-)

File: src/aaz_dev/swagger/tests/controller_tests/test_serialize_shorthand.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+def test_with_single_quote():
+    obj = {
+        "name": "monitor's"
+    }
+
+    assert serialize(obj) == '"{name:monitor\'/s}"'
+
+
+def test_with_special_characters():
+    obj = {
```
Comment (kairu-ms) [inline src/aaz_dev/swagger/tests/controller_tests/test_serialize_shorthand.py]: You can add tests for None value

File: src/aaz_dev/command/controller/shorthand.py [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    def dfs(obj):
+        if isinstance(obj, dict):
+            return "{" + ",".join(f'{k}:{dfs(v)}' for k, v in obj.items()) + "}"
+
+        elif isinstance(obj, list):
+            return "[" + ",".join(dfs(i) for i in obj) + "]"
+
+        else:
+            if obj is None:
+                return repr("null")[1:-1]  # replace None by null
```
Comment (kairu-ms) [inline src/aaz_dev/command/controller/shorthand.py]:
```suggestion
                return "null"  # replace None by null
```

**Review by kairu-ms — APPROVED** [review]
